topics/ingest-feedbacks.yaml

```diff
@@ -0,0 +1,17 @@
+topic: ingest-feedbacks
+pipeline: replays
```

Comment:
is it actually part of the replays product? isn't this just general user feedback which may or may not be associated with a replay?

Reply:
yes it's not always associated. can you elaborate on what "pipeline" is for? 

The idea was to use the same cluster/infrastructure, not cause the products are dependent, but because the same team owns them. Also to save costs

Reply:
it provides an view on what features are linked together and how data flows through the system.

As an example, if `ingest-events` is broken, then data is not going through to `events` and other related topics as well.

It's got nothing to do with which teams are working on what at sentry at any given point it time (which unfortunately seems to change every quarter).

I think you could probably just make this `errors` here, since it's moving from the shared errors topic today.

Reply:
Ok, thanks for the explanation! what does the `errors` pipeline's dependency chain look like?
I think part of the reason we want to move it off that topic is because feedback is _not_ dependent -- see motivation in https://github.com/getsentry/sentry/issues/66100. Thoughts on making a new pipeline for this topic?

Also, can we still have it on the same cluster, if it's a different pipeline? And does cluster need to be defined in some config file?

Reply:
There's quite a few things in the errors pipeline (you can search for the files in this director that contain `errors` if you want to see some example topics). New pipeline sounds fine -- you can just change it here to `user-feedback` or whatever you like here, there's nothing else to do.

Yes, the cluster topology is independent and can always be mapped to whatever kafka cluster you like per environment later.